src/DynamoCoreWpf/Extensions/ViewLoadedParams.cs

```diff
@@ -29,6 +30,11 @@ public class ViewLoadedParams : ReadyParams
         public IWatch3DViewModel BackgroundPreviewViewModel { get { return dynamoViewModel.BackgroundPreviewViewModel; } }
 
         /// <summary>
+        /// Reference to the Dynamo ViewModel to be able to execute node creation commands
+        /// </summary>
+        public IDynamoViewModel ViewModel { get { return dynamoViewModel; } }
+
```

Comment:
Instead of exposing the `IDynamoViewModel` property, I'd advise simply exposing a command framework execution endpoint on `ReadyParams`. 

Note that an organizing principle of the `Extension` framework is to incrementally expose more behavior based on the lifetime of the application. This suggestion would fit better with that concept.